metron-deployment/amazon-ec2/tasks/check-hosts.yml

```diff
@@ -16,5 +16,5 @@
 #
 ---
 - name: Wait for connectivity to host(s)
-  local_action: wait_for host={{ inventory_hostname }} state=started timeout=300 delay=10
+  local_action: wait_for host={{ inventory_hostname }} state=started port=22 timeout=300 delay=10
```

Comment:
Yes, I wanted to call this out specifically to you, but forgot.  I have been adding this "fix" in my working branch and then removing it before commits.  It has been working well for me.

Without specifying `port=22`, `wait_for` simply sleeps until the timeout.  Until now, it has just been waiting 5 minutes, every time, no matter what the state of the hosts is.

I have seen others using the following, but I think this wasn't working for you early on?  This not only waits for the port to open, but also for SSH to be ready to respond.

```
- local_action: wait_for port=22 host="{{ hostname }}" search_regex=OpenSSH delay=10
```